### DISCUSSION TITLE
Please make the web interface a 'PWA' for a nicer mobile experience

### DISCUSSION THREAD

**Discussion by GOvEy1nw** [seriousm4x/UpSnap#711] [Feature requests]
Hey, thank you for making UpSnap, it's honestly a lifesaver!

It'd be awesome if you could make it a progressive web app so that it can work better when saved to homescreens on phones etc. I believe it's fairly simple, from my ever so slight dabbling with it, but it'd make it feel more like a native app, without having to be!

**Comment by seriousm4x** [comment]
It's added!
